openstudiocore/src/utilities/filetypes/EpwFile.cpp

```diff
@@ -132,67 +132,119 @@ EpwDataPoint::EpwDataPoint(int year,int month,int day,int hour,int minute,
   setLiquidPrecipitationQuantity(liquidPrecipitationQuantity);
 }
 
-boost::optional<EpwDataPoint> EpwDataPoint::fromEpwString(std::string line)
+boost::optional<EpwDataPoint> EpwDataPoint::fromEpwString(const std::string &line)
+{
+  std::vector<std::string> list;
+  boost::split(list, line, boost::is_any_of(","));

```

Comment:
I don't really care my biggest concern is to not add any new boost dependencies.  If boost::split is a method we are using elsewhere then I guess it is ok to use here, if not we should use something that only requires C++11 (I would prefer to use C++11 over boost in any case).  

All the talk about boost vs Qt was a tangent that doesn't really apply here, C++11 should win in almost every case.